rockcraft.yaml

```
@@ -45,6 +47,21 @@ parts:
       chmod 0755 $CRAFT_PRIME/data/db
       chown -R 584788:584788 $CRAFT_PRIME/data/db
 
+      conf="etc/mongod"
+      mongo_config_file=$conf/mongod.conf
```

nit, for this and all subsequent occurrences, to prevent globbing and word splitting
```suggestion
      mongo_config_file="${conf}/mongod.conf"
```

---

This is now fixed with 656b4c6